nead/nead.py

```diff
@@ -119,3 +119,70 @@ def read(neadfile, MKS=None, multi_index=True, index_col=None):
     if('nodata' in ds.attrs.keys()): ds = ds.where(ds != ds.attrs['nodata'])
 
     return ds
+
+
+
+#%% Writes NEAD file (CSV file with NEAD formatted header)
+# Columns written in NEAD output will be the fields designated in the
+#   NEAD configuration file setting 'fields' in section 'FIELDS'
+# Paths must be absolute
+# Paths can use 'r' in front of strings with paths to designate them as a raw string if using Windows
+#   style paths with backslashes
+# Arguments:
+#   data_frame    Pandas dataframe
+#   nead_config   string with Path to NEAD configuration file, ex. r'C:\Users\Icy\gcnet\config\nead_header.ini'
+#   output_path   string with Path assigned to output NEAD file,
+#                 be sure to designate name for new NEAD file at end of string
+#                 ex.  r'C:\Icy\gcnet\output\nead\my_nead_file_name')
+from io import StringIO
+import configparser
+from pathlib import Path
+
+def read_config(config_path: str):
```

Comment:
Overall I think this is the right way to handle it - make the user define a header and then apply it. It makes the code *much* simpler. My first attempt tried to build the header from the dataframe metadata, but I found it problematic beyond the simplest cases. No changes needed here, just me commenting...